packages/react-components/icons/ForwardChevron.tsx

```diff
@@ -4,30 +4,32 @@ import Svg, { Path } from 'svgs'
 
 interface Props {
   height?: number
-  width?: number
   color?: string
 }
 
-export default class ForwardChevron extends React.PureComponent<Props> {
-  static defaultProps = {
-    height: 20,
-    width: 10,
-    color: colors.gray3,
-  }
+function ForwardChevron({ color, height }: Props) {
```

Comment:
I noticed that the BackChevron and ForwardChevron actually didn't look the same when placed side by side. so i updated them to be the ones from https://www.figma.com/file/zdK8XvxNXvTjxnM5V33pHw/Dapps-Page?node-id=1999%3A45353